cmd/importer/importer.go

```diff
@@ -40,14 +41,14 @@ func main() {
 	sec := ParseEnvVar(common.IMPORTER_SECRET_KEY, false)
 	fn := filepath.Base(ep.Path)
 	if !image.IsSupporedFileType(fn) {
-		glog.Errorf("main: unsupported source file %q. Supported types: %v\n", fn, image.SupportedFileExtensions)
+		glog.Error(Errf("unsupported source file %q. Supported types: %v\n", fn, image.SupportedFileExtensions))
 		os.Exit(1)
 	}
 	glog.Infof("main: beginning import from %q\n", ep.Path)
 	dataStream := NewDataStream(ep, acc, sec)
 	err = dataStream.Copy(common.IMPORTER_WRITE_PATH)
 	if err != nil {
-		glog.Errorf("main: copy error: %v\n", err)
+		glog.Error(err)
```

Comment:
I thought the strategy was to only call `errf` at the level where the error will be logged, and all lower level funcs just pass on the err w/o adding their name to it. If that's true then we should call `errf` here, right?

Reply:
`Err` and `Errf` have to be called at the point of failure because the stack is built starting at the caller of either func.  It's not possible to build the stack after the fact. That is, if a normal error is returned up through a call stack, we can't get the frame stack that led to it after they have all returned.